app/action/city-bike-actions.coffee

```diff
@@ -0,0 +1,17 @@
+xhrPromise = require '../util/xhr-promise'
+config     = require '../config'
+
+
+cityBikeSearchRequest = (actionContext, options, done) =>
+
+  xhrPromise.getJson(config.URL.OTP + "bike_rental", []).then((data) ->
+    actionContext.dispatch "CityBikeStationsFound", data
+    done()
+  , (err) ->
+    console.error("Failed to perform cityBikeStations search!")
+    console.error(err)
```

Comment:
These should be on the same console.error, with the err as an extra attribute, so that sentry sees them right